src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker.cs

```diff
@@ -1269,13 +1561,13 @@ protected override BoundExpression VisitExpressionWithoutStackGuard(BoundExpress
             Debug.Assert((object)resultType.Type != _invalidType.Type);
             Debug.Assert(AreCloseEnough(resultType.Type, node.Type));
 #endif
-            _callbackOpt?.Invoke(node, resultType.ToTypeSymbolWithAnnotations());
 
-            if (node.IsSuppressed || node.HasAnyErrors || !wasReachable)
+            if (ShouldMakeNotNullRvalue(node, wasReachable))
```

Comment:
>wasReachable [](start = 46, length = 12)

Can we remove this argument? All the callers of `ShouldMakeNotNullRvalue` pass `this.State.Reachable`, I think.